src/icmp/pinger.cc

```diff
@@ -65,8 +66,6 @@
 
 #include "fde.h"
 
-#define PINGER_TIMEOUT 5
```

Comment:
Done

Reply:
> PR description: change the ICMP request timeout on non-Windows systems to 10 seconds.

I fixed that claim by removing `non-` and polished.

Disclaimer: I do not know why Windows-specific timeout was set to 5s instead of the then-used 10s when Windows support was added in commit bdb741f(?).

Reply:
Oops, thank you for that.